awx/ui/src/screens/Instances/Shared/InstanceForm.js

```diff
@@ -146,17 +145,16 @@ function InstanceForm({
           description: instance.description || '',
           node_type: instance.node_type || 'execution',
           node_state: instance.node_state || 'installed',
-          listener_port: instance.listener_port || 27199,
+          listener_port: instance.listener_port,
           enabled: instance.enabled || true,
           managed_by_policy: instance.managed_by_policy || true,
-          peers_from_control_nodes: instance.peers_from_control_nodes
-            ? true
-            : !isEdit,
```

Comment:
I can't remember why we had this conditional. I just want a default value of False